DeadmansSwitch.sol

```diff
@@ -4,21 +4,32 @@ pragma solidity ^0.8.0;
 contract DeadmansSwitch {
     // TODO: Declare state variables
     // Hint: You'll need variables for the owner, beneficiary, and last check-in block
+    address owner;
+    address beneficiary;
+    uint256 lastCheckIn;
 
     // TODO: Implement constructor
     constructor(address _beneficiary) {
         // Hint: Initialize state variables
+        owner = msg.sender;
+        beneficiary = _beneficiary;
+        lastCheckIn = block.number;
     }
 
     // TODO: Implement still_alive function
     function still_alive() public {
         // Hint: Update the last check-in block
+        require(owner == msg.sender, "Owner Only!");
+        lastCheckIn = block.number;
     }
 
     // TODO: Implement release funds function
     function releaseFunds() public {
         // Hint: Check if 10 blocks have passed since last check-in
         // If so, transfer the contract balance to the beneficiary
+        require(block.number > lastCheckIn + 10, "Owner is still Alive");
```

Comment:
There is a possibility that the owner is still alive, but enough blocks has not passed. Include that also in the message